packages/core/src/pages/cp_content_schema_search.jsonc

```diff
@@ -0,0 +1,15 @@
+{
+  "search": {
+    "$extends": ["#/components/base-page-template"],
+    "$singleton": true,
+    "title": "Search Page",
+    "properties": {
+      "productGalery": {
+        "$ref": "#/components/SEO"
+      },
```

Comment:
We need to double-check this component inside an object working on the UI. I think it is working, but please double-check it!